tsweb/tsweb_test.go

```diff
@@ -496,17 +551,11 @@ func TestStdHandler(t *testing.T) {
 				t.Errorf("handler didn't write a request log")
 				return
 			}
-			errTransform := cmp.Transformer("err", func(e error) string {
-				if e == nil {
-					return ""
-				}
-				return e.Error()
-			})
-			if diff := cmp.Diff(logs[0], test.wantLog, errTransform); diff != "" {
-				t.Errorf("handler wrote incorrect request log (-got+want):\n%s", diff)
+			if diff := cmp.Diff(logs[0], test.wantLog, cmpopts.IgnoreFields(AccessLogRecord{}, "Bytes")); diff != "" {
```

Comment:
It seems like we weren't setting `Bytes` if the error handled wrote the message. Is it OK to write the error response's Bytes into this field. Surely nothing assumes this means SuccessfulBytes? 😅